dynamo.js

```diff
@@ -1,6 +1,8 @@
 const { DynamoDBClient, GetItemCommand, PutItemCommand, ScanCommand } = require("@aws-sdk/client-dynamodb");
 
-const client = new DynamoDBClient({});
+const client = new DynamoDBClient({endpoint: process.env.AWS_ENDPOINT_URL_DYNAMODB});
```

Comment:
would be nice if this wasn't necessary, but "magic" config via envs doesn't seem to work https://docs.aws.amazon.com/sdkref/latest/guide/feature-ss-endpoints.html#ss-endpoints-precedence